wikipedia-dark.user.css

```diff
@@ -1990,7 +2053,8 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
   .ns-talk .mw-body-content dl dl dl dl dl dl dl dl dl dl,
   .ns-talk .mw-body-content dl dl dl dl dl dl dl dl dl dl dl dl,
   .ns-talk .mw-body-content dl dl dl dl dl dl dl dl dl dl dl dl dl dl,
-  table:not([style*="efe"]).wikitable td:not([bgcolor]), table.prettytable td,
+  table:not([style*="efe"]):not([background-color:transparent;]).wikitable td:not([bgcolor]),
```

Comment:
I believe this was meant to be `:not([style*="background-color:transparent;"])`?